app/assets/stylesheets/frontend/reset/_reset.scss

```diff
@@ -30,3 +30,10 @@ table, caption, tbody, tfoot, thead, tr, th, td {
   font-weight: normal;
 }
 
+strong {
+  font-weight: bold;
+}
```

Comment:
What is the rational for adding this? We purposefully haven't added this in the past.